CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## 2.3.0
+
+### Features
+- **API Updates**
+  - Bumped to API version `202304` ([#64](https://github.com/singer-io/tap-linkedin-ads/pull/69))
+  - Updated API endpoints for the following streams: `campaign_groups`, `campaigns`, `creatives`
+
+- **Pagination Enhancements**
+  - Implemented cursor-based pagination for the following streams: `accounts`, `campaign_groups`, `campaigns`, `creatives`
+
+- **API Query Param Adjustments**
+  - Removed unsupported `pivot` and `pivotValue` from `fields` query parameters in Analytics API requests
+  - Incorporated these values within the tap to ensure consistency with the previous tap version
+
+- **Video Ads Stream**
+  - Added new fields ([#71](https://github.com/singer-io/tap-linkedin-ads/pull/71))
+  - Requires scope - `r_organization_social` to sync the records.
```

Comment:
Scope requirements should be mentioned in README.md and not in CHANGELOG.md unless you have made changes on tap side specific to the scope.

Reply:
done